server/src/main/java/org/elasticsearch/action/admin/indices/alias/get/TransportGetAliasesAction.java

```diff
@@ -38,32 +39,37 @@
 import java.util.Map;
 import java.util.function.Predicate;
 
-public class TransportGetAliasesAction extends TransportMasterNodeReadAction<GetAliasesRequest, GetAliasesResponse> {
+/**
+ * NB prior to 8.12 this was a TransportMasterNodeReadAction so for BwC it must be registered with the TransportService (i.e. a
+ * HandledTransportAction) until we no longer need to support {@link org.elasticsearch.TransportVersions#CLUSTER_FEATURES_ADDED} and
+ * earlier.
+ */
```

Comment:
Seems this is going to be true for all actions from the list in the task.
Is it worth to mention it once in `TransportLocalClusterStateAction` and ideally have a constant that would let us know when to remove `HandledTransportAction` base from there?

Reply:
I doubt we'll get all of them done by the time v9 is released, so I'd rather keep it a case-by-case thing.